_oasis

```diff
@@ -22,7 +22,7 @@ Flag examples
 Library webmachine
   Path:             lib
   BuildDepends:
-    calendar        (>= 2.03.2),
+    ptime           (>= 0.8.0),
```

Comment:
Ykes, this Oasis file shouldn't be here any more. Dependencies should be updated in the OPAM file and jbuild file, which it looks like you've done.

I'll send a separate PR to delete this file. Can you revert the change?

Reply:
This actually got removed in #85.